official/ubuntu_16_04/docker/laas/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# [BASH IGNORE]
+
+FROM docker.is.localnet:5000/amd/16.04:ros
+MAINTAINER Vincent Berenz <vincentberenz@tuebingen.mpg.de>
+
+# [/BASH IGNORE]
+
+###################################################################################################
+# Install robotpkg dependencies and other dependencies for dynamic graph
+###################################################################################################
+
+#[BASH UPDATE]
+# Add the robotpkg ppa. This contains the LAAS laboratory code base.
+# Typically: pinocchio, dynamic-graph, gepetto-viewer, sot-core, ...
+RUN echo "deb [arch=amd64] http://robotpkg.openrobots.org/wip/packages/debian/pub $(lsb_release -sc) robotpkg" > /etc/apt/sources.list.d/ros-latest.list
+RUN echo "deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub $(lsb_release -sc) robotpkg" >> /etc/apt/sources.list.d/ros-latest.list
```

Comment:
I just stumbled upon this issue (it already exists in the current master). This overwrites the ROS source file with the robotpkg stuff.  As a result, now new ROS packages can be installed and I am not sure if the installed ones will receive updates.  Since Ubuntu computers are set up based on the dockerfile, they are affected as well (this is how I noticed it).
Please use a separate file for robotpkg.

Reply:
@MaximilienNaveau 
@luator 

That seems indeed to be an issue. I guess using the ">>" operator (instead of the ">") would do it ?

Reply:
@MaximilienNaveau @luator 

For the sake of going on forward with this (i.e. working on corresponding bamboo job today), I will merge this, and moves this discussion to an issue